lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -292,31 +383,93 @@ class MockTimers {
     }
   }
 
-  enable(timers = SUPPORTED_TIMERS) {
+  enable(...args) {
```

Comment:
also, keep as an object so the intellisense would be better and will be ready to add more params if needed (and positional params usually are bad)

Reply:
Can do that too, I was trying to keep the usual form of JS with overloads, but option objects are far better.